heartbeat/monitors/active/http/config.go

```diff
@@ -31,10 +31,12 @@ import (
 )
 
 type Config struct {
-	URLs         []string      `config:"urls" validate:"required"`
-	ProxyURL     string        `config:"proxy_url"`
-	Timeout      time.Duration `config:"timeout"`
-	MaxRedirects int           `config:"max_redirects"`
+	URLs                 []string       `config:"urls" validate:"required"`
+	ProxyURL             string         `config:"proxy_url"`
+	Timeout              time.Duration  `config:"timeout"`
+	MaxRedirects         int            `config:"max_redirects"`
+	Response             responseConfig `config:"response"`
+	TruncateResponseBody string         `config:"truncate_response_body"`
```

Comment:
This option is nowhere documented as far as I can see.

Reply:
Apologies, yes that is leftover dead code from an early revision.